src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -727,6 +729,7 @@ static int rbd_nbd(int argc, const char *argv[])
   env_to_vec(args);
   global_init(NULL, args, CEPH_ENTITY_TYPE_CLIENT, CODE_ENVIRONMENT_DAEMON,
               CINIT_FLAG_UNPRIVILEGED_DAEMON_DEFAULTS);
+  g_ceph_context->_conf->set_val_or_die("pid_file", "");
```

Comment:
Hi Jason (and all reading this), I'm working on starting rbd-nbd with systemd. And it is best to let rbd-nbd write a pid file so that systemd can reliably determine the main process. But this config is disabled here. See [this comment](https://github.com/ceph/ceph/pull/40844#issuecomment-820168209) for more. Do you have any suggestions? thanks.

Reply:
Why not go directly and add notify/watchdog support? That would make more sense I think.